index.d.ts

```diff
@@ -11,7 +11,9 @@ declare namespace PinoSeq {
     onError?: (e: Error) => void;
   }
 
-  function createStream(config: PinoSeq.SeqConfig): Writable;
+  function createStream(config: PinoSeq.SeqConfig): Writable & { _logger: seq.Logger; flushBuffer: () => void };
```

Comment:
Will this compile? I'm not sure where the `seq.Logger` type is coming from.

Reply:
Unfortunately, `flushBuffer` is not very useful. It pushes the log events to `seq-logging` but they can still be buffered there.

Reply:
yes, I used these comments locally (in my repo):
```
    /** The underlying seq.Logger instance */
    _logger: seq.Logger
    /** Fire and forget async flush of the buffer, use _logger.flush() for a promise */
    flushBuffer: () => void
```